packages/webchannel-wrapper/package.json

```diff
@@ -16,6 +16,7 @@
   "license": "Apache-2.0",
   "devDependencies": {
     "closure-builder": "2.3.8",
+    "google-closure-library": "^20200101.0.0",
```

Comment:
We don't use version ranges in the Web SDK. Not that it matters for this versioning scheme, but it might be better to be consistent and drop the `^`.

Reply:
Oops!  I just did "npm install --save-dev" or some such...  Thanks. I'll remember this for next time!